packages/dashboards/src/lib/configurator/services/converters/timeseries/timeseries-metadata-converter.service.spec.ts

```diff
@@ -1,10 +1,12 @@
+/* eslint-disable */
 import { FormBuilder } from "@angular/forms";
 import { TimeframeService } from "@nova-ui/bits";
 
 import { IPizzagnaLayer, WellKnownProviders } from "../../../../types";
 import { IConfiguratorForm } from "../types";
 
 import { TimeseriesMetadataConverterService } from "./timeseries-metadata-converter.service";
+/* eslint-enable */
```

Comment:
it may be good to add the rule you're enabling here if possible:

```suggestion
/* eslint-enable  @typescript-eslint/no-unused-vars */
```